chapter-10-workload-identities/dbserver/Dockerfile

```diff
@@ -2,14 +2,14 @@
 # A custom Docker image for the database server with tools to enable it to detect newly downloaded SVIDs
 #
 
-FROM postgres:16-bookworm
+FROM postgres:17.2
 
 RUN apt-get update && apt-get install -y inotify-tools
 
+COPY postgresql.conf             /etc/postgresql/postgresql.conf
```

Comment:
Previously I copied these files around in some bash scripting which was a little non standard.